src/components/VHeader/VHeader.vue

```diff
@@ -1,20 +1,21 @@
 <template>
   <div
-    class="fixed flex px-8 align-center z-40 w-full bg-white"
-    :class="{ 'border-b border-dark-charcoal-20': isHeaderScrolled }"
+    class="sticky top-0 flex py-4 px-6 md:px-7 align-center justify-between z-40 w-full bg-white"
+    :class="{
+      'border-b border-dark-charcoal-20':
+        isHeaderScrolled || isFilterSidebarVisible,
+    }"
   >
     <NuxtLink to="/">
       <VLogoLoader :status="isFetching ? 'loading' : 'idle'" />
     </NuxtLink>
-    <VButton
-      v-if="!!currentOverlay"
-      variant="action-menu"
-      class="self-center"
-      @click="closeOverlay"
-    >
-      <span class="text-sr">{{ $t('modal.close') }}</span>
-      <VIcon :icon-path="closeIcon" />
-    </VButton>
+
+    <VFilterButton
+      v-if="isSearch"
```

Comment:
@sarayourfriend Are you certain this is the case? When I SSR load the page I see the filter icon, which indicates the button is rendering right away, but then the label pops in later, because the `isMinScreen` media query is false server-side. Thankfully, I added an option to the media query in #491 that allows to choose the default SSR behavior, so basically we can choose if pop-in occurs on mobile or desktop

Reply:
Yup, just double checked and took a video. It's subtle and short but there's definitely pop in. I also noticed another thing in this video is that the filter icon appears instead of the number on the initial render even when a filter is applied

https://user-images.githubusercontent.com/24264157/146492122-de1cb690-19a8-4cc9-ae48-b71b2ef7229b.mov